lib/mirage.ml

```diff
@@ -466,7 +466,8 @@ let fat_block ?(dir=".") ?(regexp="*") () =
   impl @@ object
     inherit block_conf block_file as super
 
-    method configure i =
+    method packages = Key.pure [ package "fat-filesystem" ]
```

Comment:
in this `fat_block` stanza buildtime, to make sure we have `fat` to invoke for making the filesystem.  (there's also `fat_conf` which is the section for building the fs for use by the running unikernel, which has its own dependency)